pkg/smartcontract/manifest/standard/nep17.go

```diff
@@ -14,18 +14,22 @@ var nep17 = &manifest.Manifest{
 					{Type: smartcontract.Hash160Type},
 				},
 				ReturnType: smartcontract.IntegerType,
+				Safe:       true,
```

Comment:
Is it enough? I guess `Comply` needs to be modified also to check for this.

Reply:
It was modified in ea4d14d20 .